### PR TITLE
Docker works on my end

### PR DESCRIPTION
**Pull request by nickjansen12312**
# Description

Got docker working on my end.

## Checklist

- [ ] meet all requirements of the client
- [ ] A proportionate amount of tests created
- [ ] tests passed
- [ ] peer review
- [ ] refactored and readable 
- [ ] detailed comments where need be

### CODE REVIEW

File: src/main/java/com/example/dashboard/configuration/RestConfiguration.java [hunk 1]
```diff
@@ -13,4 +13,7 @@ public WebClient webClient() {
         // Creating and returning a new instance of WebClient.
         return WebClient.create();
     }
+
+
+
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/configuration/RestConfiguration.java:18]: You can delete these three lines.

```suggestion
```

File: src/main/java/com/example/dashboard/controllers/LayoutController.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package com.example.dashboard.controllers;
 
+import com.example.dashboard.configuration.AccessPointProperties;
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/controllers/LayoutController.java:3]: You can delete this line.

```suggestion
```

File: src/main/java/com/example/dashboard/controllers/LayoutController.java [hunk 3]
```diff
@@ -16,9 +17,13 @@ public class LayoutController {
     @Autowired
     LayoutService LayoutService;
 
+    @Autowired
+    AccessPointProperties accessPointProperties;
+
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/controllers/LayoutController.java:22]: You can delete this line.

```suggestion
```

File: src/main/java/com/example/dashboard/controllers/LayoutController.java [hunk 4]
```diff
@@ -16,9 +17,13 @@ public class LayoutController {
...
     // Handling HTTP GET request to retrieve all layouts.
     @GetMapping
     public ResponseEntity<List<Layout>> getAllLayouts() {
+        accessPointProperties.getLayoutsApiUrl();
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/controllers/LayoutController.java:26]: You can delete this line.

```suggestion
```

File: src/main/java/com/example/dashboard/controllers/LayoutController.java [hunk 5]
```diff
@@ -27,6 +32,8 @@ public ResponseEntity<List<Layout>> getAllLayouts() {
     // Handling HTTP DELETE request to delete a layout by ID.
     @DeleteMapping("/{id}")
     public ResponseEntity<?> deleteLayout(@PathVariable Long id) {
+
+        accessPointProperties.getLayoutsApiUrl();
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/controllers/LayoutController.java:36]: You can delete this line.

```suggestion
```

File: src/main/java/com/example/dashboard/controllers/RuleSetsController.java [hunk 6]
```diff
@@ -1,5 +1,6 @@
 package com.example.dashboard.controllers;
 
+import com.example.dashboard.configuration.AccessPointProperties;
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/controllers/RuleSetsController.java:3]: You can delete this line.

```suggestion
```

File: src/main/java/com/example/dashboard/controllers/RuleSetsController.java [hunk 7]
```diff
@@ -11,13 +12,17 @@
 @RestController
 @RequestMapping(path="/api/ruleset")
 public class RuleSetsController {
+    @Autowired
+    AccessPointProperties accessPointProperties;
+
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/controllers/RuleSetsController.java:17]: You can delete this line.

```suggestion
```

File: src/main/java/com/example/dashboard/controllers/RuleSetsController.java [hunk 8]
```diff
@@ -11,13 +12,17 @@
...
     // Autowiring RuleSetsService to handle business logic related to rule sets.
     @Autowired
     RuleSetsService ruleSetsService;
 
     // Handling HTTP GET request to retrieve all rule sets.
     @GetMapping
     public ResponseEntity<List<RuleSet>> getAllRuleSets() {
+        accessPointProperties.getRulesApiUrl();
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/controllers/RuleSetsController.java:25]: You can delete this line.

```suggestion
```

File: src/main/java/com/example/dashboard/controllers/RuleSetsController.java [hunk 9]
```diff
@@ -26,6 +31,7 @@ public ResponseEntity<List<RuleSet>> getAllRuleSets() {
     // Handling HTTP DELETE request to delete a rule set by ID.
     @DeleteMapping("/{id}")
     public ResponseEntity<?> deleteRuleSet(@PathVariable Long id) {
+        accessPointProperties.getRulesApiUrl();
```
Comment (chris-marfia) [inline src/main/java/com/example/dashboard/controllers/RuleSetsController.java:34]: You can delete this line.

```suggestion
```

File: src/main/resources/application.properties [hunk 10]
```diff
@@ -1 +1,10 @@
 server.port=9030
+
+my.host = localhost:9030
+
```
Comment (chris-marfia) [inline src/main/resources/application.properties:4]: You can delete this since you are not using it. 

```suggestion
```

**Review by chris-marfia — CHANGES_REQUESTED** [review]
I provided suggestions on what lines of code you can delete

**Review by DemetriosDubis — APPROVED** [review]
Changes made, everything ran on our end.  

Noah was having JDK issues, unrelated to the docker issue.

**Review by nickjansen12312 — COMMENTED** [review]
looks good

**Review by chris-marfia — APPROVED** [review]

**Pull request merged**